uint8 is_outline
nav_msgs/Path path